_includes/android/users.md

```diff
@@ -522,18 +522,3 @@ ParseTwitterUtils.unlinkInBackground(user, new SaveCallback() {
   }
 });
 ```
-
-### Twitter API Calls
```

Comment:
What that removed as well? Why remove all those as user may rely on them in their implementations?

Reply:
Again, let’s not remove docs, instead, indicate why it was removed and what solution we provide, if any, and starting which version please.